mmv1/products/networkservices/LbRouteExtension.yaml

```diff
@@ -0,0 +1,187 @@
+      properties:
+        - !ruby/object:Api::Type::String
+          name: 'name'
+          description: |
+            The name for this extension chain. The name is logged as part of the HTTP request logs.
+            The name must conform with RFC-1034, is restricted to lower-cased letters, numbers and hyphens,
+            and can have a maximum length of 63 characters. Additionally, the first character must be a letter
+            and the last a letter or a number.
+          required: true
+        - !ruby/object:Api::Type::NestedObject
+          name: 'matchCondition'
+          description: |
+             Conditions under which this chain is invoked for a request.
+          required: true
+          properties:
+            - !ruby/object:Api::Type::String
+              name: 'celExpression'
+              description: |
+                A Common Expression Language (CEL) expression that is used to match requests for which the extension chain is executed.
+              required: true
+        - !ruby/object:Api::Type::Array
+          name: 'extensions'
+          description: |
+            A set of extensions to execute for the matching request.
+            At least one extension is required. Up to 3 extensions can be defined for each extension chain for
+            LbTrafficExtension resource. LbRouteExtension chains are limited to 1 extension per extension chain.
+          required: true
+          item_type: !ruby/object:Api::Type::NestedObject
+            properties:
+              - !ruby/object:Api::Type::String
+                name: 'name'
+                description: |
+                  The name for this extension. The name is logged as part of the HTTP request logs.
+                  The name must conform with RFC-1034, is restricted to lower-cased letters, numbers and hyphens,
+                  and can have a maximum length of 63 characters. Additionally, the first character must be a letter
+                  and the last a letter or a number.
+                required: true
+              - !ruby/object:Api::Type::String
+                name: 'authority'
+                description: |
+                  The :authority header in the gRPC request sent from Envoy to the extension service.
+                required: true
+              - !ruby/object:Api::Type::String
+                name: 'service'
+                description: |
+                  The reference to the service that runs the extension. Must be a reference to a backend service
+                required: true
+              - !ruby/object:Api::Type::String
+                name: 'timeout'
+                description: |
+                  Specifies the timeout for each individual message on the stream. The timeout must be between 10-1000 milliseconds.
+                  A duration in seconds with up to nine fractional digits, ending with 's'. Example: "3.5s".
+                required: true
```

Comment:
Is this field required? The API doc shows it is optional.

Reply:
updated